web/server/src/server.cc

```diff
@@ -145,16 +201,23 @@ int main(int argc, char const** argv) {
 
       auto const osm_server_file = infra_res_dir / osm_file.filename();
 
-      //copy to new location
-      try {
-          std::filesystem::copy(osm_file.c_str(), osm_server_file.c_str());
-          all_osm_paths.push_back(osm_server_file);
+      // load and copy
+      pugi::xml_document osm_data;
+      auto const error = osm_data.load_file(osm_file.c_str()); // load
+
+      const auto filtered = filter_halt(osm_data);
+      const auto fileName = osm_file.filename().replace_extension("").string();
+      halts[fileName] = extract_halt_info(filtered);
+
+
+      if (error) {
+          osm_data.save_file(osm_server_file.c_str()); // copy
       }
-      catch (const std::filesystem::filesystem_error& e) {
-          uLOG(utl::err) << e.what() << osm_server_file.filename();
+      else {
+          std::cout << "Failed to read real OSM Data. Will resume without it \n";
```

Comment:
This code appears to be from an older version of the branch OSM_Visualisation and should probably not be merged into the master since it failed in a code review before